### DISCUSSION TITLE
🎵  Add audio/sound to your projects

### DISCUSSION THREAD

**Discussion by yandeu** [enable3d/enable3d#129] [Show and tell]
I just published an audio library based on three.js's audio class.

- [@yandeu/audio - 🎵 Audio library for the Web Audio API](https://github.com/yandeu/audio#readme)

---

Enable3d is based on Three.js, why did I not included this audio library directly in enable3d?
- _Because I want everyone to be able to choose its only audio library._
- _Also, as an extern package, it is easy to inlcude this library into any website/project you want._

**Comment by N8python** [comment]
Totally epic! I really like how you keep everything as decoupled as possible.
